epic dog tamagotchi (blizzard)

gameboy333 opened the pull request:
So this game is a uhhh a game about my dog  and you feed him and you give him water and take care of him forever until the end of time. Thank you for making hack a pet so my dog could tell me to make this game.

acornitum commented:
This is cute! I like how it's the classic tamagotchi experience. I especially like the dog sprite + the shading on it.

However, I think you can do a little more with your game! 
- What if you also gave the dog an idle animation when it wants food, similar to how it has an idle animation when happy? Additionally - what if you animated the background? Maybe make leaves fall, etc! 
- Maybe add a way to play with your pet, as an additional game mechanic, like it exists in real tamagotchis? Any simple minigame would do! 
- I notice that when the countdown goes to 0 (in the output), nothing happens. Maybe a fainting animation plays or something? Currently there's not much pressure to feed the pet because nothing really happens other than it telling you to feed it
- Additionally - when I feed/water it, the countdown just infinitely goes up. Maybe set a max limit so it's realistic? 

Thanks so much for submitting! Can't wait to see when you've made those changes.

_Note: don't open a new PR, just commit changes to this current thread + send a comment when changes have been made!_

gameboy333 commented:
hello, thanks for the positive review! i've taken it into consideration and i've updated the game from it!
i made a simple minigame you can play with blizzard (the dog) where you catch balls with him, added sleeping (my equivalent of fainting as blizzard irl wanted to sleep even though we forgot to give him dinner LOL (he still got it before sleeping, though)), and more!

i couldn't really figure out how to properly add falling leaves, so i modified the spring and fall backgrounds a bit to hopefully compromise for it! as for feeding and watering it infinitely, there is and has been a limit for hunger and thirst set to 100, but i've made it so he gets hungry/thirsty more when you overfeed him, to try to discourage overfeeding

i hope this makes my game better and hopefully merged into the repo! thanks again!

acornitum commented:
This is great! The animations are sooo cute. I appreciate the timely changes too haha

Anyways, going to merge this! Fill out the form here: https://forms.hackclub.com/t/isDdJtrnrvus

Thanks so much!!
